app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(
+      author_id: current_user.id,
+      post_id: @post.id,
+      text: comment_params
+    )
+
```

Comment:
- Please there should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments.


**NB: This comment was supposed to be lined with `views/comments/new.html.erb` file, unfortunetly you did not make the file visible**